tutorials/arvo/parsing.md

```diff
@@ -164,29 +163,31 @@ of the input `nail`.
 [p=[p=1 q=2] q=[~ [p='a' q=[p=[p=1 q=2] q="bc"]]]]
 ```
 We note that `p.edg` is `[p=1 q=2]`, indicating that the next character to be
-parsed is in line 1, column 2. `q.edg` is not null, indicating that the parse
+parsed is in line 1, column 2. `q.edg` is not null, indicating that parsing
 succeeded. `p.q.edg` is `'a'`, which is the result of the parse. `p.q.q.edg` is the same as `p.edg`, which is always the case for
-`rule`s built using standard library functions when the parse succeeds. Lastly,
+`rule`s built using standard library functions when parsing succeeds. Lastly,
 `q.q.edg` is `"bc"`, which is the part of the input `tape` that has yet to be parsed.
 
-Now let's see what happens when the parse fails.
+Now let's see what happens when parsing fails.
 ```
 > =edg ((just 'b') [[1 1] "abc"])
 > edg
 [p=[p=1 q=1] q=~]
 ```
 Now we have that `p.edg` is the same as the input `hair`, `[1 1]`, meaning the
-parser has not advanced since the parse failed. `q.edg` is null, indicating that
-the parse has failed.
+parser has not advanced since parsing failed. `q.edg` is null, indicating that
+parsing has failed.
 
 Later we will use [+star](#star) to string together a sequence of `+just`s in
-order to parse multiple characters at once.
+order to parse multiple characters at once. However, it should be noted that
+`+jest`, which is covered next, ought to be used in favor of `+just`, which is
+mostly included here for pedagogical reasons.
```

Comment:
As per discussion on the comment in the previous PR, not sure if the disclaimer here is warranted.

Reply:
Yeah that makes sense. I added it in supposing that you had knowledge that `jest` was actually deprecated regardless of what the comment said.